src/main/java/org/jboss/forge/arquillian/observers/DownloadServerObserver.java

```diff
@@ -1,96 +0,0 @@
-/*
- * Copyright 2013 Red Hat, Inc. and/or its affiliates.
- *
- * Licensed under the Eclipse Public License version 1.0, available at
- * http://www.eclipse.org/legal/epl-v10.html
- */
-package org.jboss.forge.arquillian.observers;
-
-import org.jboss.forge.addon.projects.Project;
-import org.jboss.forge.addon.ui.input.UIPrompt;
-
-import javax.inject.Inject;
-
-/**
- * @Author Paul Bakker - paul.bakker.nl@gmail.com
- */
-public class DownloadServerObserver {
```

Comment:
I see most of the code was pasted to `ProfileManager`. Any particular reason why event-based, decoupled approach wasn't good for us?